images/provisions3_policy_attachment.tf

```diff
@@ -0,0 +1,9 @@
+# ------------------------------------------------------------------------------
+# Attach to the ProvisionAccount role the IAM policy that allows
+# provisioning of S3 buckets in the Images account.
+# ------------------------------------------------------------------------------
+
+resource "aws_iam_role_policy_attachment" "provisions3_policy_attachment" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonS3FullAccess"
```

Comment:
This is a very broad policy.  Can we instead create a policy that allows `s3:*` on the specific bucket ARN?  Bucket ARNs are easily constructed from the bucket name.

(Note that you may have to add a separate statement with a few actions allowed on `*` for Terraform to do its thang.  [This](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazons3.html) is your friend.)

Reply:
Excellent point- thanks for bringing this up.  Please see the changes in https://github.com/cisagov/cool-accounts/pull/47/commits/765a505196c7ecd20a96528b18c8908983bb202e, which should address your concerns.